### PR TITLE
fix issue #3752: fix url for PetaBencana.id

### PR DESCRIPTION
**Pull request by lucernae**
- upgrade keywords template
- fix url

### CODE REVIEW

**Comment by lucernae** [comment]
@timlinux @ismailsunni Please review the keywords for multi hazard

File: safe/gui/tools/peta_jakarta_dialog.py [hunk 1]
```diff
@@ -208,6 +210,39 @@ def accept(self):
...
+            if not response.ok:
+                raise Exception(request_failed_message)
+
+            result_dict = response.json()
+            with open(output_base_file_path, mode='w') as json_file:
+                json_string = json.dumps(result_dict['result'])
+                json_file.write(json_string)
+
+            # Open json file as QgsMapLayer
+            layer = QgsVectorLayer(output_base_file_path, 'flood', 'ogr',
```
Comment (timlinux) [inline safe/gui/tools/peta_jakarta_dialog.py]: pull left

File: resources/petajakarta/flood-keywords.xml [hunk 2]
```diff
@@ -175,51 +175,45 @@
...
+          <scale>
+            <gco:CharacterString/>
+          </scale>
+          <source>
+            <gco:CharacterString>PetaBencana</gco:CharacterString>
+          </source>
           <layer_geometry>
             <gco:CharacterString>polygon</gco:CharacterString>
           </layer_geometry>
+          <value_map>
```
Comment (ismailsunni) [inline resources/petajakarta/flood-keywords.xml]: Did you try to assign the keywords manually? It should be value_maps not value_map.
I suggest you to use keyword wizard if you didn't use it.

Comment (lucernae) [inline resources/petajakarta/flood-keywords.xml]: Actually, I uses keyword wizard, dude :(.

**Comment by lucernae** [comment]
@timlinux I'm not sure what Jbd is. Perhaps it stands for Jabodetabek, which is also an acronym to Jakarta-Bogor-Depok-Tangerang-Bekasi, all an area name around Jakarta. Tangerang is where the airport is.

File: resources/petajakarta/flood-keywords.xml (hunk 2 above)
Comment (ismailsunni) [inline resources/petajakarta/flood-keywords.xml]: ah sorry, it's fine. I made a typo. Shame on me. In xml it's value_map, in the code it's mapped to value_maps.
Should I change or not?

Comment (lucernae) [inline resources/petajakarta/flood-keywords.xml]: Uh, ok. I tried using value_maps now and it doesn't show up in keywords list. Maybe you should check it in your keyword wizard code? It seems to be using value_map instead. ahaha.

Comment (lucernae) [inline resources/petajakarta/flood-keywords.xml]: ah, I didn't see your comment before commenting. poor connection.
I think we should change it for consistencies.

**Comment by lucernae** [comment]
I will merge this for now, and let @myarjunar create another PR.
Note for @myarjunar, consult bang @ismailsunni for the issue about value_map. It should be changed after @ismailsunni made changes for the keywords xml.

**Pull request merged**